src/desktop/components/react/stitch_components/ReactionTooltipQuestion.tsx

```diff
@@ -1,5 +1,6 @@
 import { HelpIcon } from "@artsy/palette"
 import { Tooltip } from "v2/Components/Tooltip"
+import "v2/System/i18n/i18n"
```

Comment:
We actually wanted that this `import` would happen only [on the `Boot` component](https://github.com/artsy/force/blob/981fa970763491e8b4204a8a0cf20648101c703d/src/v2/System/Router/Boot.tsx#L31) and it shouldn't be needed anywhere else. However, the components inside the `stitch_components` directory are used in the `.jade` files for rendering some of our static pages (like `/terms`, for example), which don't rely on `Boot` - this is why we need to replicate this `import` in some of these files.

See also [this comment](https://github.com/artsy/force/pull/9449#discussion_r824933305)

Reply:
I wouldn't worry about anything that exists inside of `/desktop`; this folder is soon going to be deleted completely.

Reply:
hmm, but it's still being used right? The issue here is that if we don't make this change here, some of our existing pages crash...

Reply:
Oh i see! ok, np 👍

Reply:
Once we delete desktop this will just go away safely